README.md

```diff
@@ -185,28 +185,28 @@ ResourceSet rs = mc.resourceSet("mxwodetail").select("wonum","status").where((ne
 ResourceSet rs = mc.resourceSet("mxwodetail").select("wonum","status").where((new QueryWhere()).where("status").equalTo("APPR")).stablePaging(true).fetch();
 ```
 
-* Turn to next or previous page.
+* Turn to next or previous page:
 
 ```java 
 rs.nextPage();
 rs.previousPage();
 ```
 
-For stable paging where currently only scrolling forward is supported, a call to previousPage() would result in an error.
+For stable paging where currently only scrolling forward is supported, a call to previousPage() results in an error.
 
 * Get the ResourceSet in JSON:
 
 ```java
 JsonObject jo = rs.toJSON(); 
 ```
 
-> **note**:  we support JSON output in byte array. Try the following code,
+> **Note**:  we support JSON output in byte array. Try the following code:
 
 ```java
 byte[] jodata = rs.toJSONBytes();
 ```
 
-* Each Resource has a unique ID. It is easy to get the specific work order by it. In the following example, we try to get the Work Order (_QkVERk9SRC8xMDAw) directly.
+* Each Resource has a unique ID. It is easy to get the specific work order by it. In the following example, you can see how to get the Work Order (_QkVERk9SRC8xMDAw) directly.
```

Comment:
line 209
the _ is causing the whole line to be in italics. Is there a way that we could use a code to render the _ so that github doesn't think we're trying to italicize and just forgot the concluding _ ?